knowledge-base/binaryformatter-exception-after-upgrade-to-net5.md

```diff
@@ -0,0 +1,48 @@
+---
+title: BinaryFormatter serialization and deserialization are disabled within this application is thrown in .NET5
```

Comment:
Consider rewording the title to active voice. For example, ".NET5 projects may throw BinaryFormatter serialization and deserialization are disabled within this application"